scripts/Makefile

```diff
@@ -143,7 +143,7 @@ install:
 	$(IPROG) host-backup-restore/host-restore $(DESTDIR)$(LIBEXECDIR)
 # example/python
 	$(IDATA) examples/python/XenAPIPlugin.py $(DESTDIR)$(SITE_DIR)/
-	$(IDATA) examples/python/XenAPI.py $(DESTDIR)$(SITE_DIR)/
+	$(IDATA) examples/python/XenAPI/XenAPI.py $(DESTDIR)$(SITE_DIR)/
```

Comment:
might be useful to add a Makefile target that creates the pypi package/wheel etc. that we can then call from the specfile.

Reply:
The target should also generate the version number for the package before building it.